run_unittests.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+export PYTHONPATH=.
+
```

Comment:
hrmmm, sure this is needed,  I thought CWD was always implied.  Might be safer to do:

``sh
export PYTHONPATH=$(dirname $0):$PYTHONPATH``

or something to that effect.